src/java/org/apache/cassandra/db/ColumnFamilyStore.java

```diff
@@ -820,6 +859,10 @@ public synchronized int loadNewSSTablesWithCount()
 
     public synchronized int loadNewSSTablesWithCount(boolean assumeCfIsEmpty)
     {
+        if (assumeCfIsEmpty)
+        {
+            throw new UnsupportedOperationException("Loading new SSTables is not supported on version 2.2.18-1.164.0+.");
+        }
```

Comment:
why only throw if assumeCfIsEmpty ? wouldn't loading sstables dropped into a directory and having this method renumber this still corrupt the ancestor metadata ?

Reply:
we only run restores with assumeCfIsEmpty true, and I didn't want to break all the other tests which leverage this